PKHeX/Game/GameInfo.cs

```diff
@@ -254,7 +254,16 @@ public string[] getItemStrings(int generation, GameVersion game)
                             case GameVersion.XD:
                                 return g3xditems;
                             default:
-                                return g3items;
+                                { 
```

Comment:
Nit pick: you don't need curly braces here.  A switch statement is a glorified set of go-to statements, which aren't themselves blocks.